model/textparse/openmetricsparse.go

```diff
@@ -94,16 +94,51 @@ type OpenMetricsParser struct {
 	exemplarVal   float64
 	exemplarTs    int64
 	hasExemplarTs bool
+
+	// lines ending with _created are skipped by default as they are
+	// parsed by the CreatedTimestamp method. The skipCT flag is set to
+	// false when the CreatedTimestamp method is called.
+	skipCT bool
+}
+type openMetricsParserOptions struct {
+	// SkipCT skips the parsing of _created lines by default.
+	SkipCT bool
+}
+
+type OpenMetricsOption func(*openMetricsParserOptions)
+
+func WithSkipCT(skipCT bool) OpenMetricsOption {
+	return func(o *openMetricsParserOptions) {
+		o.SkipCT = skipCT
+	}
 }
 
 // NewOpenMetricsParser returns a new parser of the byte slice.
 func NewOpenMetricsParser(b []byte, st *labels.SymbolTable) Parser {
 	return &OpenMetricsParser{
 		l:       &openMetricsLexer{b: b},
 		builder: labels.NewScratchBuilderWithSymbolTable(st, 16),
+		skipCT:  true,
```

Comment:
false by default? Not a strong opinion, what's the rationale? Just might be surprising for existing users if somebody import that parser outside of Prometheus (likely those exists).

Reply:
set it to false by default
now one needs to use the withOptions variant to explicitly enable CT parsing